components/src/dynamo/vllm/omni/version_check.py

```
@@ -0,0 +1,61 @@
+    except ValueError:
+        return None
+
+
+def check_vllm_omni_compatibility() -> None:
+    """Raise if installed vLLM and vLLM-Omni major/minor versions differ."""
+    try:
+        from importlib.metadata import version as _pkg_version
+
+        vllm_version = _pkg_version("vllm")
+        vllm_omni_version = _pkg_version("vllm-omni")
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)